dotCMS/src/main/java/com/dotcms/graphql/datafetcher/RelationshipFieldDataFetcher.java

```diff
@@ -55,8 +56,29 @@ public Object get(final DataFetchingEnvironment environment) throws Exception {
 
             Object objectToReturn = records.doesAllowOnlyOne() ? null : Collections.emptyList();
 
-            List<Contentlet> relatedContent = contentlet.getRelated(fieldVar, user,
-                    true, isChildField, contentlet.getLanguageId(), contentlet.isLive());
+//            List<Contentlet> relatedContent = contentlet.getRelated(fieldVar, user,
+//                    true, isChildField, contentlet.getLanguageId(), contentlet.isLive());
+
+            final String query = environment.getArgument("query")!=null
```

Comment:
You can use this apache commons 3 function and code will be cleaner
```
final String query = StringUtils.defaultString(environment.getArgument("query"), "");
```

Reply:
Same for the rest of String variables.

Reply:
I like it! will do

Reply:
done